dependencies.gradle

```diff
@@ -1,7 +1,7 @@
 project.ext.versions = [
     checkstyle: '8.18',
     jacoco: '0.8.3',
-    atlas: '5.8.0',
+    atlas: '5.8.3',
```

Comment:
Do we want to do this in this PR, or mark this as DNM and wait for a dependency bump?

Reply:
I added it as atlas 5.8.3 is a certified version; Since check isn't high priority, I can wait for the next release as well. @danielduhh have we ever updated dependencies like this or we normally wait for dependency bump ?